### PR TITLE
Add Authors To Files

### PR DESCRIPTION
**Pull request by johnshamoon**

### CODE REVIEW

**Comment by johnshamoon** [comment]
[Update files with owners](https://trello.com/c/8DxLtjIq/100-update-files-with-owners)

File: src/main.py [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+Takes input from the camera to determine if the current vehicle
+should lead or follow based on whether or not there exists an ARTag
+in front of the vehicle.
+
+If an ARTag exists, the vehicle will become a follower vehicle.
+If an ARTag does not exist, the vehicle will become the leader vehicle.
+
+Authors: Zein Youssef and Steven Dropiewski
+"""
+
```
Comment (changnor) [inline src/main.py]: should there be a extra line here below the author? Its not consistent with the 2 files above.

Comment (johnshamoon) [reply src/main.py]: I think I was _going_ to do this for all of them and decided I wouldn't. Fixing now.

File: simulation/webots/controllers/follower_vehicle/Makefile [hunk 2]
```diff
@@ -11,6 +11,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
```
Comment (zeinyoussef) [inline simulation/webots/controllers/follower_vehicle/Makefile]: Remove this extra line

Comment (johnshamoon) [reply simulation/webots/controllers/follower_vehicle/Makefile]: No. This extra line is to separate the license and the author. This one is necessary.

**Pull request merged**